main.go

```diff
@@ -0,0 +1,163 @@
+package main
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/shopspring/decimal"
+)
+
+type membership struct {
+	references map[string][]string
+	members    map[string]member
+	newMembers []member
+}
+
+type transactions struct {
+	ignored   []bankTxn
+	incorrect []bankTxn
+	candidate []bankTxn
+	unmatched []bankTxn
+}
+
+type members struct {
+	paying       []member
+	unmatchedIDs []string
+}
+
+type activeMembers struct {
+	txns    transactions
+	members members
+}
+
+func newMembership(referencePath, membersPath, newMembersPath string) (*membership, error) {
+	m := membership{}
+	references, err := loadMemberReferencesFromCsv(referencePath)
```

Comment:
It's preference, but I'd prefer declaring `var err error` above here, so that you can immediately load these results into membership:

```
var err error
m := &membership{}

m.references, err = loadMemberReferencesFromCsv(referencePath)
if err != nil {
	return nil, err
}

m.members, err = loadMembershipDetailsFromCsv(membersPath)
if err != nil {
	return nil, err
}

etc.
```

Reply:
I kept trying to do this but couldn't get the syntax right. Thanks. Updated.